libs/common/src/types/mongo-db-models/facility.ts

```diff
@@ -39,7 +39,7 @@ export type Facility = {
   shouldCoverStartOnSubmission: boolean;
   coverStartDate: UnixTimestampString | Date | null;
   coverEndDate: UnixTimestampString | Date | null;
-  issueDate: UnixTimestampString | Date | null;
+  issueDate?: UnixTimestampString | Date | null;
```

Comment:
This can be undefined and null? (same with hasBeenIssuedAndAcknowledged?)

Reply:
Ooh thats a good point - I think it might always be null (on GEF deals at least, on BSS/EWCS it might be undefined)